src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -19,6 +20,12 @@ def has_object_permission(self, request: Request, view, project):
 
 
 class ProjectPerformanceIssueSettingsSerializer(serializers.Serializer):
+    n_plus_one_db_duration_threshold = serializers.FloatField(
```

Comment:
Should thresholds like these be Integer fields instead?

Reply:
Yeah, I'll convert them to integer fields. Good catch @narsaynorath